db/migrate/20150512015621_add_userstamps_to_course_users.rb

```diff
@@ -0,0 +1,14 @@
+class AddUserstampsToCourseUsers < ActiveRecord::Migration
```

Comment:
I'm not sure if this makes sense. The courseuser creator will be always.... null